processes/release_processes/README.MD

```diff
@@ -5,11 +5,15 @@
 Have a better idea about how the deployment processes should work? 
 See our "updating the process" [process](../README.MD)!
 
-## How to Publish Cromwell
+## Core Process: How to Publish and Release Cromwell
 
-![release-cromwell-version](release-cromwell-version.dot.png) 
+If you're the lucky "Release Rotation Engineer" this time, you should do the following three things:
```

Comment:
Strictly personal preference, but I prefer my mission-critical checklists to be 100% factual and humorless